pytorch_lightning/trainer/trainer.py

```diff
@@ -72,6 +72,7 @@ def __init__(self,
                  val_percent_check=1.0,
                  test_percent_check=1.0,
                  val_check_interval=1.0,
+                 no_validation=False,
```

Comment:
would be easier to set `val_check_interval` to infinity or -1? instead of adding a new parameter...